domain/domain_test.go

```diff
@@ -26,7 +27,7 @@ func Test_runTask(t *testing.T) {
 			}))
 			defer server.Close()
 
-			err := runTask(server.URL, tt.task)
+			err := RunTask(server.URL, tt.task, 2*time.Second)
```

Comment:
Oh, is this why the timeout is a parameter? Why is it shorter for the test?

Reply:
I figured that it didn't need to be very long for the test and we would rather only wait 2 seconds for the test to fail than 10 seconds.

Reply:
Maybe that point is irrelevant for tests. In any case, does it really matter that it's different in the test?

Reply:
Not sure what would cause a test to fail in a way that would hit the timeout. No problem either way, just seemed unusual.